tests/cursorid/cursorid-001.phpt

```diff
@@ -26,6 +26,7 @@ hex_dump(fromPHP(['cid' => $cursorId]));
 ===DONE===
 <?php exit(0); ?>
 --EXPECTF--
+Deprecated: MongoDB\Driver\Cursor::getId(): The method "MongoDB\Driver\Cursor::getId" will no longer return a "MongoDB\Driver\CursorId" instance in the future. Pass "true" as argument to change to the new behavior and receive a "MongoDB\BSON\Int64" instance instead. in %s
```

Comment:
Why not do `$cursor->getId(true)` above? This test is only asserting the BSON encoding, so an Int64 or CursorId makes no difference. Likewise for `cursorid-002.phpt` and `fucntional/cursorid-001.phpt`.

Reply:
This test is asserting the BSON encoding of a `CursorId` instance, and while it's technically the same as an Int64, changing this would drop test coverage of CursorId encoding.

Reply:
Oh, my mistake. I completely overlooked "cursorid" in those test names being significant.